### PR TITLE
API 4888 Immunization Schema Update

### PR DESCRIPTION
**Pull request by that-guy-is-it**
One of a few PRs that must be made for this task. This one includes schema updates for Immunization to have separate CdwId Number and Resource Code.

### CODE REVIEW

**Comment by that-guy-is-it** [comment]
@nmoisan @jtaylor-va @haidong this and the other open PR cannot be merged yet, hence the draft. Still, pinging now in case you have comments!

File: db/migration/V21_0__drop_and_create_immunization.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+drop table [App].[Immunization]
+go
+
+Create table [App].[Immunization]
+(
+	[CDWId] varchar(50) not null,
+    [CDWIdNumber] bigint null,
```
Comment (jblefkove-va) [inline db/migration/V21_0__drop_and_create_immunization.sql:7]: indentation is off

File: db/migration/V21_0__drop_and_create_immunization.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+drop table [App].[Immunization]
+go
+
+Create table [App].[Immunization]
+(
+	[CDWId] varchar(50) not null,
```
Comment (jtaylor-va) [inline db/migration/V21_0__drop_and_create_immunization.sql:6]: `CDWId` is `varchar(26)`.

File: db/migration/V21_0__drop_and_create_immunization.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    [CDWIdNumber] bigint null,
+    [CDWIdResourceCode] char(1) null,
+	[PatientFullICN] varchar(50) not null,
+	[DateUTC] datetime null,
+	[Immunization] varchar(max) null,
+	[ETLBatchId] int null,
```
Comment (jtaylor-va) [inline db/migration/V21_0__drop_and_create_immunization.sql:12]: None of these ETL columns exist in the view anymore. We also return `CVXCode char(1) null`.

Comment (nmoisan) [reply db/migration/V21_0__drop_and_create_immunization.sql:12]: `varchar(50) NULL` ***

Comment (that-guy-is-it) [reply db/migration/V21_0__drop_and_create_immunization.sql:12]: To be clear, @jtaylor-va  @nmoisan , you mean that CVXCode is a `varchar(50) NULL` ? Or did I misunderstand?

Comment (jtaylor-va) [reply db/migration/V21_0__drop_and_create_immunization.sql]: 🤦 It is `CVXCode varchar(50) null` I totally typed the wrong thing.

Comment (that-guy-is-it) [reply db/migration/V21_0__drop_and_create_immunization.sql]: @jtaylor-va No worries! Corrected it just now 👍

**Review by nmoisan — APPROVED** [review]

**Review by jtaylor-va — APPROVED** [review]

File: db/migration/V21_0__drop_and_create_immunization.sql [hunk 4]
```diff
@@ -0,0 +1,20 @@
+drop table [App].[Immunization]
+go
+
+Create table [App].[Immunization]
+(
+	[CDWId] varchar(26) not null,
```
Comment (jblefkove-va) [inline db/migration/V21_0__drop_and_create_immunization.sql:6]: spaces not tabs please

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**